### DISCUSSION TITLE
make 3.1.15 available to Ubuntu 23.xx Lunar Lobster

### DISCUSSION THREAD

**Discussion by CaliLuke** [Netatalk/netatalk#433] [Ideas]
I just installed netatalk on my Ubuntu home server with Apt Get and I noticed it pulls version 3.1.14. I checked on

https://launchpad.net/ubuntu/+source/netatalk

and I see the latest version seems to be available only for Mantic Minotaur which is still in development. Given the list of fixed bugs IMHO would be appropriate to make .15 the version installed by default unless there's incompatibilities with the previous releases of Ubuntu.

**Comment by rdmark** [comment]
Do you have a contact point at Canonical who manages the package updates?

What the downstream distributions do with netatalk releases and patches is out of the control of this project. I know for a fact that Debian unstable has a netatalk 3.1.15 package so it's up to Ubuntu when they want to adopt it.

Edit: Never mind, I see that Ubuntu has an unstable 3.1.15~ds-1ubuntu1 package too.

**Comment by rdmark** [comment]
Based on my limited understanding of the Ubuntu release process, I think they proactively bump package versions for point releases as part of the stable release cycle. 

If you want an out of cycle package version bump you need to file a Launchpad ticket as described in https://wiki.ubuntu.com/SyncRequestProcess

**Comment by rdmark** [comment]
FWIW I filed a sync request myself the other day: https://bugs.launchpad.net/ubuntu/+source/netatalk/+bug/2046609

**Comment by rdmark** [comment]
3.1.18 is in Ubuntu now! https://launchpad.net/ubuntu/+source/netatalk/3.1.18~ds-1
